l10n_it_financial_statement_eu/static/src/xml/report.xml

```diff
@@ -0,0 +1,38 @@
+<template>
+    <!-- Buttons of the Control Panel -->
+    <t
+        t-name="info.ReportAction"
+        t-inherit="web.ReportAction"
+        t-inherit-mode="extension"
```

Comment:
In questo modo viene modificato il template generico aggiungendo i pulsanti sotto, non sarebbe meglio crearne uno nuovo ereditando con `primary` dal template generico e aggiungere lì i nuovi pulsanti?
Così si eviterebbe di aggiungere questi pulsanti (nascosti grazie a `isFinancialStatementEU`) in tutti i report.

Reply:
Ci sto provando ma non riesco poi a far utilizzare al report il nuovo template (ereditato con primary) e non ho trovato un esempio. Avresti un link di un modulo dove lo fa?

Reply:
Ho fatto altre diverse prove con
```
export class FSEUReportAction extends ReportAction {
....
FSEUReportAction.template = "l10n_it_financial_statement_eu.FSEUReportButtons";
```
o simili ma non sono riuscito a trovare una soluzione funzionante

Reply:
https://discord.com/channels/753902328494424064/753902328494424070/1257956426853122098 quindi direi che su questo possiamo lasciare così